frontend/src/locale/index.ts

```diff
@@ -730,9 +731,10 @@ We look forward to reading about you and hope to see you at PyCon Italia 2024!
     "grants.form.fields.occupation.values.researcher": "Researcher",
     "grants.form.fields.occupation.values.unemployed": "Unemployed",
     "grants.form.fields.occupation.values.other": "Other",
-    "grants.form.fields.grantType": "What type of grant are you applying for?",
-    "grants.form.fields.grantType.description":
-      "Note: If you have submitted a talk/workshop proposal, you do not need to apply for a grant to receive a refund. If your proposal is accepted, we will contact you regarding the ticket refund.",
+    "grants.form.fields.grantType":
+      "Select all grant categories that apply to you",
+    "grants.form.fields.grantType.description": `You can choose more than one option if applicable.
+Note: If you have submitted a talk/workshop proposal, you do not need to apply for a grant to receive a refund. If your proposal is accepted, we will contact you regarding the ticket refund.`,
```

Comment:
We need to be more specific, we need to make sure it is clear we are only talking about the ticket. Otherwise people might think we are going to refund all their expenses

Reply:
I adjusted the text slightly